README.md

```diff
@@ -0,0 +1,144 @@
+| `service.labels`                          | Custom labels                                 | `{}`                                                    |
+| `service.clusterIP`                       | Internal cluster service IP                   | `unset`                                                 |
+| `service.loadBalancerIP`                  | IP address to assign to load balancer (if supported) | `unset`                                          |
+| `service.loadBalancerSourceRanges`        | list of IP CIDRs allowed access to lb (if supported) | `[]`                                             |
+| `serivce.externalIPs`                     | service external IP addresses                 | `[]`                                                    |
+| `persistence.enabled`                     | Use persistent volume to store data           | `false`                                                 |
+| `persistence.annotations`                 | PersistentVolumeClaim annotations             | `{}`                                                    |
+| `persistence.persistentVolumeClaim.existingClaim` | Use your own data Persistent Volume existing claim name | `unset`                               |
+| `persistence.persistentVolumeClaim.storageClass` | The milvus data Persistent Volume Storage Class | `unset`                                        |
+| `persistence.persistentVolumeClaim.accessModes` | The milvus data Persistence access modes | `ReadWriteMany`                                        |
+| `persistence.persistentVolumeClaim.size` | The size of milvus data Persistent Volume Storage Class | `50Gi`                                         |
+| `persistence.persistentVolumeClaim.subPath` | SubPath for milvus data mount               | `data`                                                  |
+| `nodeSelector`                            | Node labels for pod assignment                | `{}`                                                    |
+| `tolerations`                             | Toleration labels for pod assignment          | `[]`                                                    |
+| `affinity`                                | Affinity settings for pod assignment          | `{}`                                                    |
+| `podAnnotations`                          | Additional pod annotations                    | `{}`                                                    |
+| `podDisruptionBudget.minAvailable`        | Pod disruption minimum available              | `unset`                                                 |
+| `podDisruptionBudget.maxUnavailable`      | Pod disruption maximum unavailable            | `unset`                                                 |
+
+
+### MySQL Configuration
+
+The following table lists the configurable parameters of the mysql chart and their default values.
```

Comment:
MySQL chart?

Reply:
MySQL is Milvus subchart [(dependencies)](https://helm.sh/docs/topics/chart_best_practices/dependencies/)